landoapi/api/revisions.py

```diff
@@ -1,18 +1,100 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 from connexion import problem
+from landoapi.phabricator_client import PhabricatorClient
+
+
+def get(api_key, id):
```

Comment:
Please don't hate me but can we make this param `revision_id`?  `id` isn't descriptive and requires searching to find out what it represents